<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUserForAp</class>
 <widget class="QDialog" name="EditUserForAp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AddUserDialogForAp
{
	background-color: #FFFFFF;
}

*{
	font-family: '微软雅黑';
	font-size: 14px;

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bg" native="true">
     <property name="styleSheet">
      <string notr="true">#bg{
background-color: rgba(0,0,0,0.5);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>496</number>
      </property>
      <property name="topMargin">
       <number>164</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="container" native="true">
        <property name="minimumSize">
         <size>
          <width>458</width>
          <height>532</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>458</width>
          <height>532</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#container{
	background-color: #FFFFFF;
	border-radius: 20px;

}</string>
        </property>
        <widget class="QLabel" name="titleLabel">
         <property name="geometry">
          <rect>
           <x>181</x>
           <y>20</y>
           <width>96</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#titleLabel{
	color: black;
	font-size: 16px;
	font-weight: 1200;
	line-height: 21px;
}</string>
         </property>
         <property name="text">
          <string>编辑后台用户</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="starPhone">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>86</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starPhone{

	line-height: 19px;
	color: #FF4A3C;
	font-weight: 900;
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="emailLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>86</y>
           <width>71</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#emailLabel{
	line-height: 19px;
	color: black;
	font-weight: 900；	
}</string>
         </property>
         <property name="text">
          <string>用户邮箱</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="emailEdit">
         <property name="geometry">
          <rect>
           <x>112</x>
           <y>72</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#emailEdit{
	background-color: #F7F7F7;
	border: none;
	border-radius: 8px;
	color: #999999;
	line-height: 14px;
	padding-left: 16px;
	placeholder-text-color: #999999;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户的邮箱</string>
         </property>
        </widget>
        <widget class="QLabel" name="roleLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>154</y>
           <width>61</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#roleLabel{
	line-height: 19px;
	color: black;
	font-weight: 900；	
}</string>
         </property>
         <property name="text">
          <string>用户角色</string>
         </property>
        </widget>
        <widget class="QLabel" name="starRole">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>154</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starRole{

	line-height: 19px;
	color: #FF4A3C;
		font-weight: 900;

}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nickNameEdit">
         <property name="geometry">
          <rect>
           <x>112</x>
           <y>210</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#nickNameEdit{
	background-color: #F7F7F7;
	border: none;
	border-radius: 8px;
	color: #999999;
	line-height: 14px;
	padding-left: 16px;
	placeholder-text-color: #999999;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户昵称</string>
         </property>
        </widget>
        <widget class="QLabel" name="nickNameLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>222</y>
           <width>61</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#nickNameLabel{
	line-height: 19px;
	color: black;
	font-weight: 900；	
}</string>
         </property>
         <property name="text">
          <string>用户昵称</string>
         </property>
        </widget>
        <widget class="QLabel" name="starNick">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>222</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starNick{

	line-height: 19px;
	color: #FF4A3C;
	font-weight: 900;

}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="stateLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>286</y>
           <width>77</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#stateLabel{

	line-height: 19px;
	font-weight: 900;
	color: black;
}</string>
         </property>
         <property name="text">
          <string>用户状态</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="stateStartBtn">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>288</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#stateStartBtn{
	 color: #222222;
	

}


#stateStartBtn::checked{
	color: #3ECEFF;
	

}


#stateStartBtn::indicator{
	width: 16px;
	height: 16px;
}

#stateStartBtn::indicator::checked{
	image: url(&quot;:/image/selected.png&quot;);
}


#stateStartBtn::indicator::unchecked{
	image: url(&quot;:/image/unselected.png&quot;);
}</string>
         </property>
         <property name="text">
          <string>启用</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="stateCloseBtn">
         <property name="geometry">
          <rect>
           <x>205</x>
           <y>288</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#stateCloseBtn{
	 color: #222222;
	

}


#stateCloseBtn::checked{
	color: #3ECEFF;
	

}


#stateCloseBtn::indicator{
	width: 16px;
	height: 16px;
}

#stateCloseBtn::indicator::checked{
	image: url(&quot;:/image/selected.png&quot;);
}


#stateCloseBtn::indicator::unchecked{
	image: url(&quot;:/image/unselected.png&quot;);
}</string>
         </property>
         <property name="text">
          <string>停用</string>
         </property>
        </widget>
        <widget class="QLabel" name="bzLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>330</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#bzLabel{
	font-weight:900;
	line-height: 19px;
	color: #222222;
}
</string>
         </property>
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="commentEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>360</y>
           <width>398</width>
           <height>84</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#commentEdit{
	backgroud-color: #F7F7F7;
	border: none;
	outline: none;
	border-radius:8px;
	color: #222222;
	/* padding: 14px 16px 14px 16px; */
	background-color: white;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入内容...</string>
         </property>
        </widget>
        <widget class="QLabel" name="wordCount">
         <property name="geometry">
          <rect>
           <x>385</x>
           <y>415</y>
           <width>41</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#wordCount{
	color: #999999;
	


}</string>
         </property>
         <property name="text">
          <string>12</string>
         </property>
        </widget>
        <widget class="QComboBox" name="roleComboBox">
         <property name="geometry">
          <rect>
           <x>112</x>
           <y>140</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#roleComboBox{
	background-color: #F7F7F7;
	border: none;
	border-radius: 8px;
	color: #999999;
	line-height: 14px;
	padding-left: 16px;
	placeholder-text-color: #999999;
}

#roleComboBox::drop-down{
	border: none;
	width: 33px;
}

#roleComboBox::down-arrow{
	width: 12;
	height: 12;
	background-color: transparent;
	image: url(&quot;:/image/more.png&quot;);
}

#roleComboBox QAbstractItemView{
	outline: 0px;
}

#roleComboBox QAbstractItemView::item{
	color: #222222;
	padding-left: 16px;
	background-color: #FFFFFF;
	height: 30px;
}

#roleComboBox QAbstractItemView::item:selected{
	background-color: #409CE1;
	color: #FFFFFF;
	


}


</string>
         </property>
         <property name="placeholderText">
          <string>请选择用户角色</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancelBtn">
         <property name="geometry">
          <rect>
           <x>114</x>
           <y>474</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#cancelBtn{
	background-color: #FFFFFF;
	border: 1px solid #DEDEDE;
	 border-radius: 8px;
	color: #999999;
	font-size: 16px;
}</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
        <widget class="QPushButton" name="submitBtn">
         <property name="geometry">
          <rect>
           <x>244</x>
           <y>474</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#submitBtn{
	background-color: #3ECEFF;
	border: 1px solid #3ECEFF;
	 border-radius: 8px;
	color: #FFFFFF;
	font-size: 16px;
}</string>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
        <zorder>titleLabel</zorder>
        <zorder>starPhone</zorder>
        <zorder>emailLabel</zorder>
        <zorder>emailEdit</zorder>
        <zorder>starRole</zorder>
        <zorder>roleLabel</zorder>
        <zorder>nickNameEdit</zorder>
        <zorder>starNick</zorder>
        <zorder>stateLabel</zorder>
        <zorder>stateStartBtn</zorder>
        <zorder>stateCloseBtn</zorder>
        <zorder>bzLabel</zorder>
        <zorder>commentEdit</zorder>
        <zorder>wordCount</zorder>
        <zorder>roleComboBox</zorder>
        <zorder>nickNameLabel</zorder>
        <zorder>cancelBtn</zorder>
        <zorder>submitBtn</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
